web_src/js/features/colorpicker.js

```diff
@@ -1,10 +1,12 @@
-export async function createColorPicker($els) {
-  if (!$els || !$els.length) return;
+import $ from 'jquery';
+
+export async function createColorPicker(els) {
+  if (!els?.length) return;
```

Comment:
```suggestion
  if (!els.length) return;
```

`els` is from either `$` or `querySelectorAll`, it couldn't be null.